### PR TITLE
docs(adr): [proposal] use gnu make to centralise all tasks

### PR DESCRIPTION
**Pull request by esciara**

### CODE REVIEW

File: docs/adr/XXXX-use-gnu-make-to-centralise-all-tasks.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# Use GNU Make to centralise all tasks
+
+* Status: proposed
+* Date: 2020-05-04
+
+## Context and Problem Statement
+
+We want to be able to centralise all tasks in a single tool, to be call during the
```
Comment (slequeux) [inline docs/adr/XXXX-use-gnu-make-to-centralise-all-tasks.md]: `to be call` => `to be called`

File: docs/adr/XXXX-use-gnu-make-to-centralise-all-tasks.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    * Like GNU Make, an emphasis on minimal boilerplate for common patterns 
+      and the ability to run multiple tasks in a single invocation.
+* Bad, because:
+  * It is not a de facto standard for Python.
+  * It has a large number of 
+    [unresolved and untriaged issues](https://github.com/pyinvoke/invoke/issues)
+    (202 at time of writing) and a large number of 
+    [opened Pull Requests](https://github.com/pyinvoke/invoke/pulls), some dating back
+    2014, which suggests a slack in maintenance.
+  * it does not seem to have a large enough community.
```
Comment (slequeux) [inline docs/adr/XXXX-use-gnu-make-to-centralise-all-tasks.md]: `it` => `It`

File: docs/adr/XXXX-use-gnu-make-to-centralise-all-tasks.md (hunk 1 above)
Comment (esciara) [reply docs/adr/XXXX-use-gnu-make-to-centralise-all-tasks.md]: done

File: docs/adr/XXXX-use-gnu-make-to-centralise-all-tasks.md (hunk 2 above)
Comment (esciara) [reply docs/adr/XXXX-use-gnu-make-to-centralise-all-tasks.md]: done

**Pull request merged**